lib/cardano-sl.cabal

```diff
@@ -156,7 +156,7 @@ library
                       , constraints
                       , containers
                       , contravariant
-                      , cryptonite >= 0.23 && <= 0.24
+                      , cryptonite >= 0.23 && <= 0.25
```

Comment:
Since we had a specific dependency on a github version, I suspect this should be:
```
                      , cryptonite == 0.25.*
```

Reply:
I don't think the constraint make sense, it was just because it was unreleased at the time and there was a need for a specific commit for a runtime issue (not an API problem). it's already pinned to .25 through nix and stack.